### DISCUSSION TITLE
MOOSE team job opening

### DISCUSSION THREAD

**Discussion by GiudGiud** [idaholab/moose#21570] [News]
Hello!

We have a job opening for a computational scientist on the MOOSE team!

All the details about the posting: [inl.taleo.net](https://inl.taleo.net/careersection/inl_external/jobdetail.ftl?job=18073&tz=GMT-06%3A00&tzname=America%2FBoise)

If you enjoy open-source software that benefits thousands of researchers worldwide, please apply!

Guillaume for the MOOSE team

**Comment by faisal-bhuiyan** [comment]
Hi @GiudGiud. This is Faisal Bhuiyan - I believe I will be a good fit for this role and applied via the above link. Looking forward to hearing back and please feel free to ask questions!

**Comment by permcody** [comment]
This posting is now closed. We are conducting interviews over the next few weeks. Thanks to all who applied!
